tests/pages/services/ServiceActions-cy.js

```diff
@@ -51,16 +51,6 @@ describe('Service Actions', function () {
 
   context('Destroy Action', function () {
 
-    context('Framework', function () {
```

Comment:
*Side Note: Let's please make sure to always run the integration test locally to make sure we're not merging something without adjust the respective test as we otherwise never come to the point where the integration tests finally work.* /cc  @dcos/frontend